v2.0/admin-ui-node-map.md

```diff
@@ -0,0 +1,185 @@
+---
+title: Node Map 
+toc: false
+---
+
+The **Node Map** view visualizes the geographical configuration of a multi-regional cluster by plotting the node locations on a world map. The **Node Map** also provides real-time cluster metrics, with the ability to drill down to individual nodes to monitor and troubleshoot the cluster health and performance. 
+
+<div id="toc"></div>
+
+{{site.data.alerts.callout_info}}The <b>Node Map</b> is an <a href="enterprise-licensing.html">enterprise-only</a> feature. However, you can <a href="https://www.cockroachlabs.com/pricing/request-a-license/">request a trial license</a>  to try out the <b>Node Map</b> view. {{site.data.alerts.end}}
+
+## Why Use the Node Map View
+
+The **Node Map View** helps you monitor the cluster health and performance, ensure effective resource utilization, and troubleshoot cluster issues:
+
+### Monitor Cluster Health and Performance
+
+The **Node Map** view provides:
+
+- A high-level graphical overview of how the cluster is configured, which helps you understand how the available hardware is servicing your users. 
+- Important cluster health metrics, such as Capacity and CPU utilization, QPS, and uptime.
+
+### Ensure Effective Resource Utilization
+
+The **Node Map** view helps you:
+
+- Confirm that the database is balanced and you are utilizing the resource effectively. This helps you provision hardware and plan ahead for scale.
+- Ensure that storage capacity is balanced across the disks available to the extent that they match the zone configs.
+- Check if there resource congestion anywhere (for example certain nodes going beyond capacity).
+
+### Troubleshoot the Cluster
+
+The **Node Map** view helps you identify and preempt any scenarios that might cause downtime, service issues, or otherwise negatively impact cluster health. The **Node Map** view helps you:
+
+- Monitor disk utilization to know if you need to add more storage capacity.
+- Identify dead nodes in the cluster.
+- Be informed about unavailable or under-replicated ranges.
+- Confirm that configuration changes were persisted to the cluster.
```

Comment:
:-/ not exactly

Reply:
@couchand Which part? Should I just drop the "Confirm that configuration changes were persisted to the cluster" part?

Reply:
I think for now we should drop it.  That's something that would be good to add, but there's not really much configuration you can see come through the map (besides the config of the map itself).